tests/Pyxsim/xe.py

```diff
@@ -0,0 +1,98 @@
+from Pyxsim.xmostest_subprocess import call, call_get_output
```

Comment:
Needs a copyright comment and a comment referencing the license.  See the Source check failures in the Jenkins run.

Reply:
added